backend/src/main/java/ee/tlu/forum/model/User.java

```diff
@@ -0,0 +1,47 @@
+package ee.tlu.forum.model;
+
+import com.sun.istack.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.Collection;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "users") // 'user' is a reserved keyword and cannot be used
+public class User extends BaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @NotNull
+    @Column(unique = true)
+    private String username;
+
+    @NotNull
+    private String email;
```

Comment:
I'm thinking the email should also be unique? So they can't create multiple accounts with the same email address?